skills/morph-ppt/reference/styles/dark--premium-navy/build.sh

```
@@ -0,0 +1,314 @@
+officecli set "$OUTPUT" '/slide[4]/shape[4]' --prop preset=roundRect --prop x=15.5cm --prop y=3.5cm --prop width=8cm --prop height=8cm --prop opacity=0.35 --prop fill=$NAVY
+officecli set "$OUTPUT" '/slide[4]/shape[5]' --prop x=28cm --prop y=12cm --prop width=4cm --prop height=4cm --prop opacity=0.25
+officecli set "$OUTPUT" '/slide[4]/shape[6]' --prop x=25cm --prop y=4cm --prop width=3cm --prop height=3cm --prop opacity=0.15
+officecli set "$OUTPUT" '/slide[4]/shape[7]' --prop x=30cm --prop y=2cm
+officecli set "$OUTPUT" '/slide[4]/shape[8]' --prop x=24cm --prop y=16cm
+
+# Update title to metrics
+officecli set "$OUTPUT" '/slide[4]/shape[9]' --prop text="Performance Metrics" --prop size=36 --prop align=left --prop x=1.2cm --prop y=0.8cm --prop width=20cm --prop height=2.5cm
+officecli set "$OUTPUT" '/slide[4]/shape[10]' --prop text="FY2025 Annual Results" --prop size=16 --prop color=$GRAY --prop align=left --prop x=1.2cm --prop y=2.8cm --prop width=12cm --prop height=1.2cm
+
+# Show metrics (reuse card shapes)
+officecli set "$OUTPUT" '/slide[4]/shape[11]' --prop text="$128M" --prop size=64 --prop x=2.4cm --prop y=5.5cm --prop width=10cm --prop height=3.5cm
```

Shellcheck (0.11.0)

[error] Braces are required for positionals over 9, e.g. ${10}.

(SC1037)